pkg/info/info.go

```diff
@@ -36,19 +44,23 @@ func getEksctlVersion() string {
 
 // getKubectlVersion returns the kubectl version
 func getKubectlVersion() string {
-	cmd := exec.Command("kubectl", "version", "--short", "--client")
+	cmd := exec.Command("kubectl", "version", "--client", "--output", "json")
 	out, err := cmd.CombinedOutput()
 	if err != nil {
 		return fmt.Sprintf("error : %v", err)
 	}
 
-	values := strings.FieldsFunc(string(out), func(c rune) bool {
-		return unicode.IsSpace(c)
-	})
-	if len(values) == 0 {
+	var info kubectlInfo
+
+	if err := json.Unmarshal([]byte(string(out)), &info); err != nil {
```

Comment:
`out` is already a `[]byte`, the conversion is not required. 
```suggestion
	if err := json.Unmarshal(out, &info); err != nil {
```